src/Core/Models/OrganizationConnectionConfig.cs

```diff
@@ -0,0 +1,7 @@
+namespace Bit.Core.Models
+{
+    public class BillingSyncConfig
```

Comment:
Rename file to match class, please.

Maybe move to an `OrganizatinConnectionConfigs` model subfolder, too